html/modules/custom/bc_dc/tests/src/Functional/BcDcFunctionalTest.php

```diff
@@ -536,6 +536,13 @@ public function test(): void {
     // Page has ISO dates.
     $this->isoDateTest();
 
+    // Anonymous has access to download csv for Metadata record when file has been uploaded
+    $this->drupalGet('node/2/download/columns/csv');
+    $this->assertSession()->statusCodeEquals(200);
```

Comment:
It would be good to test that the contents of this file are as expected.

Reply:
Not sure how to test this

Reply:
You can probably use `::assertSession()->responseContains()`.

Reply:
Used this assert